components/bl-audio-player-component/README.md

```diff
@@ -1,25 +1,51 @@
 # Audio Player
 
-This is a component for Backendless [UI-Builder](https://backendless.com/developers/#ui-builder) designer. You will have a styled player with the set of your tracks. It supports one or multiple audio sources and has all functions you can think of.
-
-While the player can be configured with multiple settings it is easy in use and provides users with smooth experience.
+The Audio Player component provides basic functionality to play audio files. 
```

Comment:
It could be streams too.

Reply:
Please describe the format of the audio stream value, how does it look like? What property expects this audio stream value?

Reply:
https://cast.radiogroup.com.ua/nrj

Reply:
Added